src/wallet/wallet.cpp

```diff
@@ -405,7 +397,7 @@ std::shared_ptr<CWallet> CreateWallet(WalletContext& context, const std::string&
     }
 
     // Descriptor support must be enabled for an external signer wallet
-    if ((wallet_creation_flags & WALLET_FLAG_EXTERNAL_SIGNER) && !(wallet_creation_flags & WALLET_FLAG_DESCRIPTORS)) {
+    if (wallet_creation_flags & WALLET_FLAG_EXTERNAL_SIGNER) {
```

Comment:
In 8fbb1c9657ef034aa179655a0869db45be2aaac8 "wallet, refactor: Remove Legacy warnings and errors"

That's wrong and causing a CI failure. I think this check is unneeded.

Reply:
Yeah, my bad, also de error description was clear. Just found another case, I'll fix this soon. Thanks!

Reply:
Sorry, having said that, the error description needs to be updated mentioning that external signer should be enabled instead of descriptor wallet.